telco-examples/edge-clusters/cpu-manager/README.md

```diff
@@ -0,0 +1,45 @@
+# Example of Edge Cluster with CPU Manager
+
+## Introduction
+
+This is an example to demonstrate how to deploy an edge cluster with cpu manager enabled in kubernetes for Telco using SUSE ATIP and the fully automated directed network provisioning.
```

Comment:
- Kubernetes
- I'm not sure what you mean exactly for "directed", but maybe "automated" is sufficient?

Reply:
At one point, direct network provisioning was the term we had decided to use to be distinct from Elemental's "phone home provisioning". It looks like we now call it "BMC automated deployments" for the quickstart page so maybe standardize on that?

Reply:
We discussed this some time ago for 3.0 and agreed to go with the term "directed" in the product docs for the CAPI/Metal3 automated flow https://documentation.suse.com/suse-edge/3.1/html/edge/id-suse-edge-documentation.html#id-directed-network-provisioning

In the context of this repo "BMC automated" is probably fine, but we should be cautious of introducing inconsistencies in the docs, and if we want to revisit the preferred terminology there it will require a broader discussion with the team and adjacent stakeholders (for example it potentially impacts various marketing materials and other customer-facing resources)

Reply:
> We discussed this some time ago for 3.0 and agreed to go with the term "directed" in the product docs for the CAPI/Metal3 automated flow https://documentation.suse.com/suse-edge/3.1/html/edge/id-suse-edge-documentation.html#id-directed-network-provisioning

I was not aware of any of these details, thanks 👍

Reply:
`Kubernetes` suggest changed